.github/workflows/upload-cli.yml

```
@@ -20,12 +20,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # build and publish in parallel: linux/amd64, linux/arm64, linux/s390x, darwin/amd64, darwin/arm64
+        # build and publish in parallel: linux/amd64, linux/arm64, linux/s390x, linux/ppc64le, darwin/amd64, darwin/arm64
         goos: [linux, darwin]
-        goarch: [amd64, arm64, s390x]
+        goarch: [amd64, arm64, s390x, ppc64le]
         exclude:
           - goos: darwin
-            goarch: s390x
+            goarch: [s390x, ppc64le]
```

actionlint (1.7.12)

[error] value ["s390x", "ppc64le"] in "exclude" does not match in matrix "goarch" combinations. possible values are "amd64", "arm64", "s390x", "ppc64le"

(matrix)